apps/core/lib/core/application.ex

```diff
@@ -4,8 +4,15 @@ defmodule Core.Application do
   use Application
 
   def start(_type, _args) do
+    configure_mnesia()
     children = [Core.Repo]
     opts = [strategy: :one_for_one, name: Core.Supervisor]
     Supervisor.start_link(children, opts)
   end
+
+  defp configure_mnesia() do
+    :mnesia.create_schema([node()])
+    Application.ensure_started(:mnesia)
```

Comment:
Mnesia process is started manually. It means that the supervisor won't handle failures. This is important because once the underlying Mnesia process will die, the supervisor won't start it again. As the result, all API's endpoints that read or write events will fail with unhandled HTTP 500 (for example 
https://github.com/smt116/elixir-umbrella-sample-application/pull/4/files#r221667057).

Mnesia should started as the `Core`'s child.

Reply:
btw. you don't have `:ok = Application.ensure_started(:mnesia)` so this instruction could fail.